src/modules/app/VirtualMouseModule.js

```diff
@@ -21,11 +21,14 @@ export class VirtualMouseModule extends Events {
     this.patchEvents = patchEvents;
   }
 
-  update(e) {
+  update(e, customX, customY) {
     const rect = this.canvas.getBoundingClientRect();
 
-    this.mouse.x = ((e.clientX - rect.left) / (rect.right - rect.left)) * 2 - 1;
-    this.mouse.y = -((e.clientY - rect.top) / (rect.bottom - rect.top)) * 2 + 1;
+    const x = customX || e.clientX;
+    const y = customY || e.clientY;
+
+    this.mouse.x = ((x - rect.left) / (rect.right - rect.left)) * 2 - 1;
```

Comment:
this was b/c alex is helping me with my project and I was multiplying x by 2. in any case, I want to improve vmouse so it can accept function params so that user can overload how x/y are constructed using `movementX` and `movementY` from pointer lock mouse event